test/standalone/c_compiler/build.zig

```diff
@@ -23,15 +28,30 @@ pub fn build(b: *Builder) void {
     exe_c.setBuildMode(mode);
     exe_c.setTarget(target);
     exe_c.linkLibC();
+    exe_c.enable_wine = is_wine_enabled;
+    exe_c.enable_qemu = is_qemu_enabled;
+    exe_c.enable_wasmtime = is_wasmtime_enabled;
+    exe_c.enable_darling = is_darling_enabled;
```

Comment:
I wonder why I decided to make this per-artifact instead of global to Builder. I think I'd like to explore changing this (myself, I will make changes to master branch), as a prerequisite to merging this PR.